test cases/common/262 generator chain/meson.build

```diff
@@ -0,0 +1,19 @@
+project('Generator Chain', 'c')
+
+stage1_exe = find_program('stage1.py')
+
+stage2_exe = find_program('stage2.py')
+
+stage1_gen = generator(stage1_exe,
+  output : '@PLAINNAME@.inter',
+  arguments : ['@INPUT@', '@OUTPUT@'])
+
+stage2_gen = generator(stage2_exe,
+  output : '@PLAINNAME@.c',
+  arguments : ['@INPUT@', '@OUTPUT@'])
+
+out = stage2_gen.process(stage1_gen.process(files('data.txt')))
```

Comment:
You don't need  `files` here. It should only be used when you define source files in one place and then use it somewhere later.

Reply:
Done